### ISSUE TITLE
Parser changing content.

### ISSUE THREAD

**Issue by iammattmartin** [jstanden/cerb#699]
We have to send logs to customers. i.e

```
[2018-08-07 14:15:19] NOTICE[614] chan_sip.c: Registration from '<sip:x-xxx@sip-x.xxx.net>' failed for 'xxxx:5060' - No matching peer found
```

But cerb is converting that to

```
[2018-08-07 14:15:19] NOTICE[614] chan_sip.c: Registration from '[sip:x-xxx@sip-x.xxx.net](mailto:sip:x-xxx@sip-x.xxx.net)' failed for 'xxxx:5060' - No matching peer found
```

Is there a way to stop this because along with the double spacing of indented lines starting with - (after sending), the plain text looks awful and should really be "as sent".

**Comment by jstanden** [comment]
It sounds like this was sent in HTML/Markdown, and not plaintext, right?

If so, I'd recommend sending the log entries in code format (which converts them to a PRE/CODE block).  You can wrap them in `~~~` (triple tilde).

**Comment by iammattmartin** [comment]
It was indeed.

I'll give that a bash although would probably be nice to stop it doing that for HTML emails as an option on the message instead of having to remember to use formatting options (as not everyone will I don't think).

**Comment by iammattmartin** [comment]
The spacing between lines is still off though when changed from HTML to text. It's just a bit annoying as the text version is what we show to the user in their account so we often have to go and edit the post after sending to make it look nice.

**Comment by iammattmartin** [comment]
We've just popped a pull request in for the double spacing issue. Might be worth reviewing if you can as that makes the saved content consistent with what you edit and then send.

**Comment by jstanden** [comment]
This should be fixed in 9.6. We now preserve the original Markdown for the `text/plain` part (like any sane developer would).
